ext/node/lib.rs

```diff
@@ -73,6 +74,11 @@ impl NodePermissions for AllowAllNodePermissions {
 pub type NpmResolverRc = deno_fs::sync::MaybeArc<dyn NpmResolver>;
 
 pub trait NpmResolver: std::fmt::Debug + MaybeSend + MaybeSync {
+  /// Gets the state of npm for the process.
+  fn get_npm_process_state(&self) -> String {
+    String::from("")
```

Comment:
If this isn't called, it might be better to use `unimplemented!` or `unreachable!` here.